src/components/Article/Article.tsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from 'react-router-dom'
+import './Article.css';
+
+interface ArticleProps {
+  title: string
+  image: string
+  abstract: string
+  caption: string
+}
+
+const Article = ({ title, image, abstract, caption }: ArticleProps): JSX.Element => {
+
+  return (
+    <section className="single-article-container">
+      <Link to="/feed">
+        BACK
+      </Link>
+      <figure>
+        <img src={image} alt={caption} className="single-article-image"/>
+        <figcaption className="single-article-caption">{caption}</figcaption>
+      </figure>
+      <h2 className="single-article-title">{title}</h2>
+      <p>{abstract}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+      <p>{placeholderText}</p>
+    </section>
```

Comment:
Maybe add a back button to the end of the article so that after reading the article, the user doesn't need to scroll all the way back to the top.

```suggestion
      <Link to="/feed">
        BACK
      </Link>
    </section>
```